data_handler/handlers/loan_states/zklend/events.py

```diff
@@ -180,10 +180,11 @@ def process_accumulators_sync_event(self, event: pd.Series) -> None:
     def process_deposit_event(self, event: pd.Series) -> None:
         # The order of the values in the `data` column is: `user`, `token`, `face_amount`.
         # Example: https://starkscan.co/event/0x036185142bb51e2c1f5bfdb1e6cef81f8ea87fd4d777990014249bf5435fd31b_3.
-        user = add_leading_zeros(event["data"][0])
-        token = add_leading_zeros(event["data"][1])
+        event_data = ZklendDataParser.parse_borrowing_event(event["data"])
+        user = add_leading_zeros(event_data.user)
+        token = add_leading_zeros(event_data.token)
 
-        face_amount = decimal.Decimal(str(int(event["data"][2], base=16)))
+        face_amount = decimal.Decimal(str(int(event_data.face_amount, base=16)))
```

Comment:
move this logic `decimal.Decimal(str(int(event_data.face_amount, base=16)))` to Model level